cli/cli.go

```diff
@@ -4,17 +4,32 @@ import (
 	"bytes"
 	"encoding/json"
 	"fmt"
+	cdx "github.com/CycloneDX/cyclonedx-go"
 	"github.com/jfrog/build-info-go/build"
 	"github.com/jfrog/build-info-go/utils"
+	"github.com/pkg/errors"
 	clitool "github.com/urfave/cli/v2"
+	"os"
+)
+
+const (
+	formatFlag = "format"
 )
 
 func GetCommands(logger utils.Log) []*clitool.Command {
+	flags := []clitool.Flag{
+		&clitool.StringFlag{
+			Name:  formatFlag,
+			Usage: "[Optional] If specified, the build-info will be printed in another format. Supported formats: 'cdx/xml' or 'cdx/json'` `",
```

Comment:
If specified, the build-info will be printed in another format. Supported formats: 'cdx/xml' or 'cdx/json'
-->
Set to convert the build-info to a different format. Supported values are cyclonedx/xml and cyclonedx/json